copyright

```diff
@@ -407,12 +407,29 @@ Copyright: Various
 License: public-domain
  Based on public domain sounds taken from freesound.org.
 
-Files: sounds/heavy rocket hit.wav
+Files: sounds/heavy?rocket?hit.wav
 Copyright: Copyright Mike Koenig
 License: CC-BY-SA-3.0
 Comment: Taken from http://soundbible.com/1467-Grenade-Explosion.html
 
-Files: sounds/explosion huge.wav
+Files: sounds/missile?hit.wav
+Copyright: Copyright "Nbs Dark"
+License: public-domain
+Comment: Taken from https://freesound.org/people/Nbs%20Dark/sounds/94185/
+
+Files: sounds/torpedo?hit.wav
```

Comment:
@Pointedstick Whose copyright is this? I ask because as it turns out not having a Copyright field is a violation of the spec.

Reply:
Just saw on that other PR that you don't know. So `Copyright: Public Domain` should be enough I think. Sorry for the unnecessary ping.